From: "Dr. Git" <drgitx@gmail.com>
Date: Sat, 24 Jul 2021 16:37:11 -0300
Subject: arm64: dts: qcom: sdm845-oneplus-common: enable headset jack
 detection

Adds headset jack and button detection voltage thresholds for the OnePlus 6/6T.
---
 arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi b/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
index c007db1e..2f6c15b 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
@@ -773,6 +773,10 @@ &wcd9340 {
 	vdd-tx-supply = <&vreg_s4a_1p8>;
 	vdd-rx-supply = <&vreg_s4a_1p8>;
 	vdd-io-supply = <&vreg_s4a_1p8>;
+
+	qcom,mbhc-buttons-vthreshold-microvolt = <112000 220000 437000 600000 600000 600000 600000 600000>;
+	qcom,mbhc-headset-vthreshold-microvolt = <1700000>;
+	qcom,mbhc-headphone-vthreshold-microvolt = <50000>;
 };
 
 &wifi {
